src/docs/index.md

```diff
@@ -214,6 +215,15 @@ and learn about other series on our [videos][] page.
 
 <iframe style="max-width: 100%" width="560" height="315" src="//player.bilibili.com/player.html?aid=55794948&cid=97538589&page=1" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+First up, why use Flutter? What makes it different than other
+app frameworks?
+
+首先你需要知道，为何我们要使用 Flutter？ 是什么让它与众不同？
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/l-YO9CmaSUM" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe><br>
```

Comment:
@wanglin6600 https://www.youtube.com/embed/l-YO9CmaSUM upload to Bilibili, please.

Reply:
https://www.bilibili.com/video/av68736077/